### DISCUSSION TITLE
After deploy on vercel keystatic is not working

### DISCUSSION THREAD

**Discussion by 7Hemant** [Thinkmill/keystatic#524] [Q&A] [answered]
I have created the Keystatic project and made some changes according to the documentation for deployment on Git and connecting with Vercel, but it’s not working as expected. When I navigate to the route ‘site-name/keystatic’, it doesn’t take me to the login page as mentioned in the documentation. I’m not sure why this issue is occurring. Has anyone else faced the same issue?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Are you confident the environment variables have been added to the Vercel? Also, have you redeployed the site after adding those variables? They only take effect on the next deployment. This got me personally a couple of times already 😅

**Reply by 7Hemant** [reply]
Thanks, I checked all the credentials and set up a new project again. Now it’s working.

**Reply by simonswiss** [reply]
Nice one — glad to hear you're unblocked 🎉
